apps/src/musicMenu/MusicMenu.jsx

```diff
@@ -82,6 +82,22 @@ const optionsList = [
     type: 'string',
     description: 'Set a specific key by name (i.e. "C", "C#", "D", etc)',
   },
+  {
+    name: 'skip-controls',
+    type: 'radio',
+    values: [
+      {value: 'disabled', description: 'Disable skip controls.'},
+      {value: 'enabled', description: 'Enable skip controls.'},
```

Comment:
super nit: we use `true` and `false` for other values in the menu.  `enabled` and `disabled` are definitely nice, but should we be consistent with whichever we choose?  One factor might be what works best in the context of levelbuilder parameters.

Reply:
Oh good call, I'll make this `true`/`false` for consistency.